review_practice/react-app/src/containers/SearchBar/ModalBody/PersonnelFilter.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useContext } from 'react';
+import PersonnelController from 'components/PersonnelController';
+
+import { SearchRoomContext } from 'pages/SearchRoom';
+
+export default ({ buttonState, setButtonState }) => {
+    const {
+        searchOption: {
+            searchOptions: { personnel },
+        },
+        dispatchSearchOption,
+    } = useContext(SearchRoomContext);
+
+    const adultState = useState(personnel && personnel.adult ? personnel.adult : 0);
+    const childrenState = useState(personnel && personnel.children ? personnel.children : 0);
+    const infantState = useState(personnel && personnel.infant ? personnel.infant : 0);
```

Comment:
```suggestion
    const adultState = useState(personnel?.adult ||  0);
    const childrenState = useState(personnel?.children || 0);
    const infantState = useState(personnel?.infant || 0);
```
`optionalChanning`을 사용하면 조금 더 명시적으로 표현할 수 있습니다!